docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -501,6 +494,22 @@ we should:
 - Acknowledge data limitations and proxy measures used.
 - Call for better data to be collected.
 
+To understand this, we can look at our figure of hypothetical data sources 
+from above. There is now an additional row with how we would include 
+this data in our analysis. You can see the first two data sources would both be 
+undifferentiated sex/gender. This is due to ambiguity between sex and gender in 
+the question asked. 
+
+The latter two data source more clearly include sex, although one is sex assigned 
+at birth and the other is sex identity. Therefore if you included both data source 
+3 and 4, you might label this as undifferentiated sex factors. 
```

Comment:
True! I think it would be clear that 'sex' is an ambiguous term that, in the case of this example, includes a conflation of sex assigned at birth and sex identity